src/sentry/api/endpoints/project_details.py

```diff
@@ -258,7 +258,12 @@ def put(self, request, project):
                         'detail': ['The new team is not found.']
                     }, status=400
                 )
-            old_team_id = project.team_id
+            # TODO(jess): update / deprecate this functionality
+            try:
+                old_team_id = project.teams.values_list('id', flat=True)[0]
```

Comment:
Doesn't `old_team_id` need to be set to `None` in the `except` case?

Reply:
nm, it's set to that above.

Reply:
yeah